src/pages/index.tsx

```diff
@@ -7,14 +7,17 @@ import {
   clear,
   incrementByAmount,
   changeCounter,
+  getTestData,
 } from "../store/reducer/counterSlice";
 import HeaderLayout from "../layouts/HeaderLayout";
 import { Button, Box, Flex, Heading, Text } from "@chakra-ui/react";
 import ChangeCourtSize from "@/components/ChangeCourtSize";
+import { AppDispatch } from "@/store";
```

Comment:
There is an import store inside of the _app.js file already. That should make the redux store global on all pages and components.